packages/shared/src/components/FeedItemComponent.tsx

```diff
@@ -133,6 +154,19 @@ export default function FeedItemComponent({
     ranking,
   );
 
+  const { shouldUseFeedLayoutV1 } = useFeedLayout();
+  const PlaceholderTag = getPlaceHolder(useList, shouldUseFeedLayoutV1);
+
+  let postTypeToTag = PostTypeToTag;
+  let articlePostCard = ArticlePostCard;
+  let EffectiveAdTag = AdTag;
```

Comment:
I'd try to avoid this `let` basically and maybe we can opt for a better mapping function that has 2 dimensions? (version/cards)